src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -261,6 +261,91 @@ void jit_equal_emitter::register_table_entries() {
     push_arg_entry_of("one", 0x3f800000, true);
 }
 
+/// ELU ///
+jit_elu_emitter::jit_elu_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                 dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                 const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    const auto elu = std::dynamic_pointer_cast<ov::op::v0::Elu>(node);
+    if (elu == nullptr) {
+        OV_CPU_JIT_EMITTER_THROW("Can't cast to ov::op::v0::Clamp");
+    }
+    alpha = static_cast<float>(elu->get_alpha());
+
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, node);
+}
+
+jit_elu_emitter::jit_elu_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                 dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                 const float alpha,
+                                 const ov::element::Type exec_prc) : jit_emitter(host, host_isa, exec_prc), alpha(alpha) {
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, exec_prc);
+}
+
+size_t jit_elu_emitter::get_inputs_count() const { return 1; }
+
+size_t jit_elu_emitter::get_aux_vecs_count() const {
+    return std::max(exp_emitter->get_aux_vecs_count() + 1ull, 2ull);
```

Comment:
As I understand, we need 1 more aux_gprs for p_table. But can you please elaborate why we need 1 more vec aux_vecs? I mean why it's not just:
`return std::max(exp_emitter->get_aux_vecs_count(), 2ull);`

Reply:
> As I understand, we need 1 more aux_gprs for p_table. But can you please elaborate why we need 1 more vec aux_vecs? I mean why it's not just: `return std::max(exp_emitter->get_aux_vecs_count(), 2ull);`

Source and destination can be the same register. As result, we need additional auxiliary register to store original input value before `exp_emitter` code emittion.

Reply:
Cool! Goot to know that exp_emitter supports shared src and dst registers.

Minor: In jit_exp_emitter::emit_isa, perhaps we should save vmm_src (e.g., to vmm_aux1) right after the defination of vmm_mask https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L305, if vmm_src.getIdx() == vmm_dst.getIdx(). Otherwise the calculation of vmm_mask https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L313 won't make sense, because the values in vmm_src have already changed. What do you think?

Reply:
Good point, fixed, thanks. But we definitely need tests for that for both platforms. let's do this later: CVS-141109